js/react_native/e2e/android/app/build.gradle

```diff
@@ -207,6 +213,8 @@ dependencies {
     androidTestImplementation 'androidx.test:rules:1.4.0'
 
     implementation project(':onnxruntime-react-native')
+    // specify ORT dependency here so it can be found in libs flatDir repository
+    implementation "com.microsoft.onnxruntime:onnxruntime-mobile:latest.integration@aar"
```

Comment:
this was done so that the e2e test picks up the local version of the .aar if one is provided. before, it was getting the released package. seems a bit clunky but I don't know enough about gradle to come up with a better way

FYI @hanbitmyths